app/views/books/isbn.html.slim

```diff
@@ -0,0 +1,6 @@
+= form_for :book, url: '/api/v1/books/isbn', remote: true, html: {class: ['flex-column', 'new-form'], id: 'isbn-form'} do |f|
```

Comment:
Add handlers for AJAX success and AJAX error. On success just show the fields received.